### DISCUSSION TITLE
Adding BigWig files on MultiQuantitative track

### DISCUSSION THREAD

**Discussion by rshakhb10** [GMOD/jbrowse-components#3559] [Q&A] [answered]
Hi! I am trying to add bigwig files under MultiQuantitative Track under "bigWigs" keyword on json. The files are stored locally on a persistent storage. However, jbrowse fails to load them... 
Any insight on what I should be rather doing differently?  
I am doing it like here: https://jbrowse.org/jb2/docs/config_guides/multiquantitative_track/#:~:text=%22MultiWiggleAdapter%22%2C-,%22,-bigWigs%22%3A.  But instead of URLs these are bigwig file paths stored locally. 
Thank you!

**Comment by cmdcolin** [comment, marked as answer]
@rshakhb10 this is actually a oversight in the design of the system, you have to refer to absolute URLs when using the bigWigs array currently. if you want to use relative URLs you can use the subadapters array

mentioned here https://github.com/GMOD/jbrowse-components/issues/3430#issuecomment-1377597684

**Reply by cmdcolin** [reply]
I'll make an issue from this discussion so this has a proper issue to track! thanks for reporting

**Comment by rshakhb10** [comment]
Hmmm, but it looks like that neither relative or absolute paths work... it only seemed to work when I switched to the subadaptors

**Reply by cmdcolin** [reply]
what error did you see? just for reference, this is an example of an example of a config using absolute paths https://github.com/GMOD/jbrowse-components/blob/d9e5ecc2f56a0eb6d329bb89f77eeaebdec3da71/test_data/config_demo.json#L3969-L4001 but if subadapters works, might be best for now anyways since it supports the relative paths

**Reply by rshakhb10** [reply]
yes I am doing something similar but instead of URLs, I point to the locally stored files like /path/to/the/file/file.bigWig and it doesn't work... is it only intended for urls? It gives a 404 error, file cannot be found

**Reply by cmdcolin** [reply]
yes, technically everything is a URL to jbrowse. this is basically because jbrowse has no "server side"  so it can't access e.g. absolute paths on the file system, only URLs so a URL like /path/to/the/file/file.bigWig would be resolved as http://yourserver.com/path/to/the/file/file.bigWig

**Reply by rshakhb10** [reply]
okay,  yeah it can't fetch it, despite that the file is present...
